src/k2/tso/client/Client.cpp

```diff
@@ -48,14 +49,41 @@ void TSOClient::_registerMetrics() {
 }
 
 seastar::future<> TSOClient::start() {

```

Comment:
I guess we can leave it for now, but I don't think we need the dist-based TSO client anymore so this init sequence can go away.